.travis.yml

```diff
@@ -16,10 +16,12 @@ python:
   - 3.6
   - 3.7
   - 3.8
+  - 3.9
+  - 3.10-dev
```

Comment:
Unfortunately, travis-ci does not yet have a GA python 3.10 build. This works for now, I think. Tracking this post for updates: https://travis-ci.community/t/add-python-3-10/12220